java/api/src/main/java/com/exadel/frs/core/trainservice/cache/ModelStatisticCacheEntry.java

```diff
@@ -0,0 +1,13 @@
+package com.exadel.frs.core.trainservice.cache;
+
+import lombok.Getter;
+
+@Getter
+public class ModelStatisticCacheEntry {
+
+    private long requestCount;
+
+    public void incrementRequestCount() {
+        requestCount++;
```

Comment:
it is not thread safe.
The idea of how to make it threat safe:
1. Remove this class, and store Integer instead
2. https://stackoverflow.com/a/47586624 - use cache.asMap().compute